inference-platforms/archgw/arch_config.yaml

```diff
@@ -8,7 +8,9 @@ listeners:
     timeout: 30s
 
 llm_providers:
-  - model: ollama/qwen3:0.6b
+  # ollama/ in the model name forces using the ollama interface even though we
```

Comment:
If the main reason is to be able to put a proxy or similar behind this proxy, I'm not sure if it's worth having this workaround. But will leave it to you